v2/src/lib.rs

```diff
@@ -700,17 +697,24 @@ impl Storage for InfluxDbStorage {
                     continue;
                 }
             };
-            let value = Value::new(payload).encoding(encoding);
+            let value = Value::new(payload, encoding);
             result.push(StoredData { value, timestamp });
         }
         Ok(result)
     }
 
     //putting a stub here to be implemented later
     async fn get_all_entries(&self) -> ZResult<Vec<(Option<OwnedKeyExpr>, Timestamp)>> {
+        tracing::warn!("!!! get_all_entries is NOT implemented yet !!!"); // See : https://github.com/ZettaScaleLabs/zenoh-backend-influxdb/pull/4
         tracing::warn!("called get_all_entries in InfluxDBv2 storage");
         let mut result: Vec<(Option<OwnedKeyExpr>, Timestamp)> = Vec::new();
-        let curr_time = zenoh::time::new_reception_timestamp();
+        let curr_time = match std::num::NonZeroU128::new(1u128).map(new_timestamp) {
```

Comment:
Add comment like "TODO: pass real zid from the session instead" if not fix it right now